src/ripple/app/tx/impl/Escrow.cpp

```diff
@@ -435,6 +450,16 @@ EscrowFinish::doApply()
             return ter;
     }
 
+    // Remove escrow from recipient's owner directory, if present.
+    if (ctx_.view ().rules().enabled(fix1523) && (*slep)[~sfAltOwnerNode])
+    {
+        TER const ter = dirDelete(ctx_.view(), true,
+            (*slep)[sfAltOwnerNode], keylet::ownerDir((*slep)[sfDestination]),
+            k.key, false, false, ctx_.app.journal ("View"));
```

Comment:
The owner deletion [checks `page==0`](https://github.com/ripple/rippled/pull/2212/files#diff-42cae634a40574e7b3ba49cf6b25972bR448) for the `bSoft` argument (second to last) of `dirDelete`.  Is that important to mirror here?

Reply:
No: that's legacy code (that should have never been there for things like escrow).